android/cli/commands/_buildModule.js

```diff
@@ -58,7 +58,7 @@ AndroidModuleBuilder.prototype.migrate = async function migrate() {
 	const manifestModuleAPIVersion = this.manifest.apiversion;
 	const manifestTemplateFile = path.join(this.platformPath, 'templates', 'module', 'default', 'template', 'android', 'manifest.ejs');
 	let newVersion = semver.inc(this.manifest.version, 'major');
-	this.tiSdkVersion = cliSDKVersion;
+	this.tiSdkVersion = cliSDKVersion.includes('.GA') ? cliSDKVersion : cliSDKVersion + '.GA';
```

Comment:
The Titanium CLI stores the SDK name in the `cli.sdk`:

```
{
  name: '12.4.0.GA',
  manifest: {
    name: '12.4.0.GA',
    version: '12.4.0',
    moduleAPIVersion: { iphone: '2', android: '4' },
    timestamp: '7/17/2024 18:43',
    githash: 'a5451fc00d',
    platforms: [ 'android' ]
  },
  path: 'C:\\ProgramData\\Titanium\\mobilesdk\\win32\\12.4.0.GA',
  platforms: {
    android: {
      path: 'C:\\ProgramData\\Titanium\\mobilesdk\\win32\\12.4.0.GA\\android'
    }
  },
  type: 'ga',
  version: '12.4.0'
}
```

So we can do:

```suggestion
	this.tiSdkVersion = this.cli.sdk.name;
```